.gitattributes

```
@@ -28,6 +28,9 @@
 *.attrmap binary diff=hex
 *.tilemap binary diff=hex
 
+# Declare files that will always have CRLF line endings on checkout.
+pokecrystalvc.patch.template text eol=crlf
```

For decent syntax highlighting of the patch file's `; comments` and `[headings]`:
```suggestion
pokecrystalvc.patch.template text eol=crlf linguist-language=INI
```